vuepress/docs/v7.1/docs/create/component-gen-tech.md

```diff
@@ -3,8 +3,7 @@
 When generating a microservice using the Entando Component Generator there are a number of technical choices that need to be made
 
 * Microservice code
-  * If you're interested in using Quarkus in your Entando app, contact us on the [Forum](https://forum.entando.org/) or [Community Slack](https://join.slack.com/t/entandocommunity/shared_invite/zt-g609owdv-2K~YRh8zrI6lqlWo4aFWUw) and we'll help you get started.
-  * If you're considering non-Java based microservices, [here is a definition of an Entando plugin](../curate/bundle-details.md#microservices-specifications), including information on the runtime contract required to use them. 
+  * If you're considering non-Java based microservices, here are the [specifications of an Entando microservice](../curate/bundle-details.md#microservices-specifications), including information on the runtime contract required to use them. 
```

Comment:
sooo... it should be Java-based with a hyphen, and the negative of that in the form non-Java-based is a definite no... "microservices that aren't java-based" to avoid this?